packages/flags/src/index.ts

```diff
@@ -1,4 +1,9 @@
-export type Flag = 'PERSISTENT_EXAMPLE' | 'PROFILE_EDITING';
+export type Flag =
+  | 'PERSISTENT_EXAMPLE'
+  | 'EDIT_PROFILE_SKILLS'
+  | 'EDIT_PROFILE_QUESTIONS'
+  | 'EDIT_PROFILE_WORKS'
+  | 'EDIT_PROFILE_REST';
```

Comment:
Not turning on yet because we're not sending invites yet, and I think one of the edits (role on team) may not be stable